### PR TITLE
Clipboard System

### PR DESCRIPTION
**Pull request by CodedStingray**

### CODE REVIEW

File: src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package net.codedstingray.worldshaper.util.world;
+
+import net.codedstingray.worldshaper.util.vector.vector3.Vector3ii;
+import org.bukkit.block.data.BlockData;
+
+public record PositionedBlockData(Vector3ii position, BlockData blockData) {
+
+}
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java:8]: add newline at the end

File: src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package net.codedstingray.worldshaper.util.world;
+
+import net.codedstingray.worldshaper.util.vector.vector3.Vector3ii;
+import org.bukkit.block.data.BlockData;
+
+public record PositionedBlockData(Vector3ii position, BlockData blockData) {
+
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java]: remove newline in record

**Review by ZidArgs — CHANGES_REQUESTED** [review]

File: src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package net.codedstingray.worldshaper.util.world;
+
+import net.codedstingray.worldshaper.util.vector.vector3.Vector3ii;
+import org.bukkit.block.data.BlockData;
+
+public record PositionedBlockData(Vector3ii position, BlockData blockData) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java:12]: missing javadoc

File: src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java (hunk 2 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java]: removed

File: src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java (hunk 3 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/util/world/PositionedBlockData.java:12]: added

**Review by ZidArgs — APPROVED** [review]

**Pull request merged**